pypif_sdk/interop/mdf.py

```diff
@@ -0,0 +1,229 @@
+from pypif_sdk.readview import ReadView
+from pypif.obj.common import Value, ProcessStep
+from citrination_client import PifQuery, DatasetQuery, SystemQuery, Filter
+from ..util.citrination import get_client
+from pypif.pif import dumps
+
+import re
+
+
+def query_to_mdf_records(query=None, dataset_id=None, acl=None):
```

Comment:
It looks like dataset_id is silently ignored if query is present, right? What about adding it to the query?

Reply:
The intention is for one of the two to be specified.  I think it makes more sense to raise a value error.